dotcom-rendering/src/components/LoopVideoPlayer.tsx

```diff
@@ -143,6 +149,9 @@ export const LoopVideoPlayer = forwardRef(
 					tabIndex={0}
 					onError={onError}
 					css={videoStyles(width, height)}
+					data-link-name={`gu-video-loop-${
```

Comment:
Why `video-loop` instead of `loop-video` or `looping-video`? (also lines 169 & 192). Is this an ophan convention? The latter would be consistent with the `data-component` value

Reply:
good question: this follows the convention set by the youtube player tracking https://github.com/search?q=repo%3Aguardian%2Fdotcom-rendering%20gu-video-youtube&type=code

You're right thought that to align with this we are not consitent with the data component - i'll update this in line with the current convention